mcp/mcp-server/src/main/java/software/amazon/smithy/java/mcp/server/McpServer.java

```diff
@@ -339,6 +339,8 @@ private Map<String, Tool> createTools(Map<String, Service> services) {
                                 operationName,
                                 schema))
                         .inputSchema(createJsonObjectSchema(operation.getApiOperation().inputSchema(), new HashSet<>()))
+                        .outputSchema(
```

Comment:
I think we should do this based on the MCP version. This was only added in version `2024-11-05`